extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_links.php

```diff
@@ -18,7 +18,21 @@
 		<div class="wds-dropdown__content wds-global-navigation__dropdown-content <?= $dropdownRightAligned ? 'wds-is-right-aligned' : ''; ?>">
 			<ul class="wds-list">
 				<?php foreach ( $model['links'] as $link ): ?>
-					<li><?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkDropdown', [ 'model' => $link ] ); ?></li>
+					<li>
+						<?php if ( $link['type'] === 'link-authentication' ) {
+							echo $app->renderView(
+								'DesignSystemGlobalNavigationService',
+								'linkAuthentication',
+								[ 'model' => $link ]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "model" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)